content/tutorials/channel-enumeration-rest.textile

```diff
@@ -14,24 +14,22 @@ Channel enumeration, as decribed above, is part of the Channel Status API, and c
 
 h2(#permissions). Step 2 - Setting the right permissions on your API key
 
-In order to be able to use the enumeration feature, you'll need to ensure that the @Channel Metadata@ permission is enabled on your API key. "Permissions":/core-features/authentication/#capabilities-explained (or privileges or capabilities, used interchangeably) restrict your API key from being used to performing specific actions only, such as publish-and-subscribe or publish-only, etc. These privileges can be set on your API key via your account dashboard. Navigate to the 'API Keys' tab of your dashboard as shown in the image below and click on the 'settings' button against an existing API key that you'd like to use or create a new one.
+In order to be able to enumerate channels, you'll need to ensure that the @Channel Metadata@ permission is enabled on your API key. "Privileges":/core-features/authentication/#capabilities-explained (sometimes referred to as capabilities or permissions) restrict your API key to specific operations such as publish-and-subscribe or publish-only. These privileges can be set on your API key via your account dashboard. Navigate to the 'API Keys' tab of your dashboard as shown in the image below and click on the 'settings' button on your existing API key that you'd like to use, or create a new one.
```

Comment:
`publish-and-subscribe` and `publish-only` I don't believe are specific permissions? If not, maybe remove the `-` between `publish-and-subscribe` and bolden each actual permission.